tests/kernels/test_prefix_prefill.py

```diff
@@ -17,12 +18,14 @@
 
 
 @pytest.mark.parametrize("num_heads", NUM_HEADS)
+@pytest.mark.parametrize("num_queries_per_kv", NUM_HEADS)
```

Comment:
Should this parameter be NUM_QUERIES_PER_KV?

Reply:
Fixed in #3246.